### ISSUE TITLE
google-map when used inside core-pages renders transparently on top of all pages

### ISSUE THREAD

**Issue by rajasharan** [GoogleWebComponents/google-map#107]
sample screenshot:
![image](https://cloud.githubusercontent.com/assets/299522/6097952/5957aae4-af9d-11e4-8d58-e4ef549c53dc.png)

sample code the way i'm using core-pages:

```html
<div flex layout vertical>
  <core-pages flex>
    <info-page></info-page>
    <map-page></map-page>
  </core-pages>
</div>
```

code for map-page:

```html
<polymer-element name="map-page" flex layout vertical>
  <template>
    <style>
      div { position: relative; }
   </style>
  
   <div flex>
    <google-map latitude="37.779" longitude="-122.3892" minZoom="9" maxZoom="11" fit>
      <google-map-marker latitude="37.779" longitude="-122.3892" draggable></google-map-marker>
   </google-map>
  </div>
 </template>
. . .
</polymer-element>
```

(I'm using all the flex's because otherwise the height would be set to 0px - sorry still learning my way around CSS)

**Comment by ebidel** [comment]
It actually looks like the map is _behind_ all pages, not on top of it.

**Comment by rajasharan** [comment]
yes behind all pages :)

I couldn't import `http://googlewebcomponents.github.io/google-map/components/google-map/google-map.html` in jsbin because of CORS issue. Is there a public cdn i can use?

**Comment by ebidel** [comment]
How about posting an demo.html file and a bower.json file for me to install locally?

**Comment by rajasharan** [comment]
thanks for looking into this, below are the index and bower files.

index.html
```html
<!doctype html>
<html>
    <head>

        <script src="bower_components/webcomponentsjs/webcomponents.js"></script>

        <link href="bower_components/polymer/polymer.html" rel="import">

        <link rel="import" href="bower_components/core-pages/core-pages.html">
        <link rel="import" href="bower_components/paper-button/paper-button.html">
        <link rel="import" href="bower_components/paper-item/paper-item.html">
        <link rel="import" href="bower_components/google-map/google-map.html">

    </head>
    <body fullbleed layout vertical>

        <polymer-element name="my-element">
            <template>
                <style></style>
                <paper-button center on-click="{{next}}">NEXT</paper-button>
                <div flex one></div>
                <div flex three layout horizontal>
                    <core-pages id="pages" flex>
                        <div> PAGE 1 </div>
                        <div> PAGE 2 </div>
                        <div> PAGE 3 </div>
                        <div class="map" flex>
                            <google-map fit></google-map>
                        </div>
                    </core-pages>
                </div>
                <div flex one></div>
            </template>

            <script>
                Polymer('my-element', {
                    next: function() {
                        this.$.pages.selected = (this.$.pages.selected + 1 ) % this.$.pages.children.length;
                    }
                });
            </script>

        </polymer-element>

        <my-element flex layout vertical></my-element>
    </body>
</html>
```

bower.json
```js
{
  "name": "google-map-bug-107",
  "version": "0.0.0",
  "dependencies": {
    "core-elements": "Polymer/core-elements#^0.5.1",
    "paper-elements": "Polymer/paper-elements#^0.5.1",
    "polymer": "Polymer/polymer#^0.5.1",
    "google-map": "GoogleWebComponents/google-map#~0.4.0"
  },
  "devDependencies": {
    "web-component-tester": "Polymer/web-component-tester#^2.0.0"
  }
}
```

**Comment by emfluenceindia** [comment]
Hi,
I have installed Polymer using Yeoman. But inside bower_components folder I cannot see the folder "google-map". The only Google related folder I can see is "google-code-prettify".

Is this the reason when I placing a <google-map> component in my html, it is not rendering? If so, how can I get those missing folders installed inside bower_components?

Also, I am getting some JS errors (attached). Are they somehow related to google-map?

Thanks for your help!

Subrata Sarkar

![js-error-1](https://cloud.githubusercontent.com/assets/11072678/6438940/cf476f52-c0f3-11e4-9a3c-ee9edc3ba60c.png)

![js-error-2](https://cloud.githubusercontent.com/assets/11072678/6438941/cf8f9926-c0f3-11e4-887a-2f7cc22ab867.png)

**Comment by atotic** [comment]
I've successfully used google-map inside iron-pages, and neon-animated-pages. Fixed in 1.0.
